params.go

```diff
@@ -75,24 +75,24 @@ type ListParams struct {
 	// key or query the state of the API.
 	Context context.Context `form:"-"`
 
-	EndingBefore string   `form:"ending_before"`
-	Expand       []string `form:"expand"`
-	Filters      Filters  `form:"*"`
-	Limit        int64    `form:"limit"`
+	EndingBefore *string   `form:"ending_before"`
+	Expand       []*string `form:"expand"`
+	Filters      Filters   `form:"*"`
+	Limit        *int64    `form:"limit"`
 
 	// Single specifies whether this is a single page iterator. By default,
 	// listing through an iterator will automatically grab additional pages as
 	// the query progresses. To change this behavior and just load a single
 	// page, set this to true.
 	Single bool `form:"-"` // Not an API parameter
```

Comment:
shouldn't this one be `*bool` too?

Reply:
I was kind of split on this one ... the reason I didn't change it is that it's not really an API parameter, more like an internal pagination option for the library's iterator. The downside is that it may seem a little inconsistent, but it does have the advantage is that we will never care about the unset versus `false` states. What do you think?